istio/tests/common.py

```diff
@@ -393,3 +388,31 @@
     'istio.galley.istio.networking.gateways',
     'istio.galley.istio.authentication.meshpolicies',
 ]
+
+PREVIOUSLY_BLACKLISTED_TAGS = [
```

Comment:
Could you leave a comment explaining what time "previously" refers to? Imagine someone reading this in a year with no context for the story of this development.